python/examples/ingestion_with_csv/multi_csv/main.py

```diff
@@ -0,0 +1,124 @@
+import csv
+import os
+from datetime import datetime
+from pathlib import Path
+from typing import Dict, List, Tuple
+
+from dotenv import load_dotenv
+from sift_py.grpc.transport import SiftChannelConfig, use_sift_channel
+from sift_py.ingestion.channel import ChannelConfig, ChannelDataType, double_value, empty_value
+from sift_py.ingestion.config.telemetry import TelemetryConfig
+from sift_py.ingestion.flow import FlowConfig, FlowOrderedChannelValues
+from sift_py.ingestion.service import IngestionService
+
+
+def parse_csv(
+    data: List[Dict], telemetry_config: TelemetryConfig
+) -> List[FlowOrderedChannelValues]:
+    flows: List[FlowOrderedChannelValues] = []
+
+    flow = telemetry_config.flows[0]  # Packed into a single flow for this example
+    flow_name = flow.name
+
+    all_timestamps: List = []
+    for channel in data:  # Extract all timestamps from all channels
+        all_timestamps += channel.keys()
```

Comment:
You could also just get away with `all_timestamps = [ts for ts in channel.keys()`]?

Reply:
Yeah this would be nicer with a comprehension, I like it 👍